requirements.yml

```diff
@@ -16,7 +16,7 @@
 
 - name: openmicroscopy.omero-server
   src:  https://github.com/openmicroscopy/ansible-role-omero-server.git
-  version: 2.0.0-m1
+  version: 2.0.1
```

Comment:
Now `2.0.2` is out, should this be bumped as I understand the playbook was deployed with the role changes integrated in https://github.com/openmicroscopy/ansible-role-omero-server/pull/22?